cinder/autogenerated_client.go

```diff
@@ -0,0 +1,1821 @@
+package cinder
```

Comment:
Corrected.

Just FYI, it looks like no other files have the copyright. We might want a separate PR to correct that.